test/fork/NFTDescriptor.t.sol

```diff
@@ -48,11 +48,11 @@ abstract contract NFTDescriptor_Fork_Test is Fork_Test {
     /// @dev Given enough fuzz runs, all the following scenarios will be fuzzed:
     /// - Multiple values of streamId.
     function testForkFuzz_TokenURI_LockupDynamic_V2_0(uint256 streamId) external loadDeployments_V2_0 {
-        streamId = bound(streamId, 1, lockupDynamic.nextStreamId() - 1);
+        streamId = _bound(streamId, 1, lockupDynamic.nextStreamId() - 1);
```

Comment:
Why `_bound` and not `bound`?

Reply:
Doesn't emit console logs.

https://github.com/foundry-rs/forge-std/blob/e4aef94c1768803a16fe19f7ce8b65defd027cfd/src/StdUtils.sol#L60-L63